docs/examples.md

```diff
@@ -7,13 +7,14 @@ pipx install --python python3.8 pycowsay
 pipx install git+https://github.com/psf/black
 pipx install git+https://github.com/psf/black.git@branch-name
 pipx install git+https://github.com/psf/black.git@git-hash
+pipx install git+ssh://<username>@<private-repo-domain>/<path-to>/<package.git>
```

Comment:
```suggestion
pipx install git+ssh://<username>@<private-repo-domain>/<path-to-package.git>
```

I don’t think there’s really much reason to split these?

Reply:
The reason why I split it that I was not sure if it is clear that you have to include the `package.git` in the path. But do what you think is best.